### PR TITLE
Twitter blog roll and hyperlink styles changed

### PR DESCRIPTION
**Pull request by syedjafer**
Twitter blog roll and hyperlink styles changed

### CODE REVIEW

File: css/pysangamam.css [hunk 1]
```diff
@@ -37,7 +38,7 @@ h1#title:hover span:after {
 }
 
 h1#title:hover span.mobile-span:after {
-  font-size: 55px;
+  font-size: 55px;`
```
Comment (astronomersiva) [inline css/pysangamam.css:41]: Looks like a typo

File: css/pysangamam.css [hunk 2]
```diff
@@ -18,6 +18,7 @@ body, .main {
 
 h1 {
   font-weight: bold;
+  font-size: 80px;
```
Comment (astronomersiva) [inline css/pysangamam.css:21]: Which element are you trying to target with this?

File: css/pysangamam.css [hunk 3]
```diff
@@ -208,3 +209,8 @@ svg.icon-link {
     text-align: center;
   }
 }
+
+a{
+  color : #218c74;
+  text-decoration: None;
+}
```
Comment (astronomersiva) [inline css/pysangamam.css:216]: You will also need to style the pseudo classes for the anchor tag to make this work properly. As in, add the styles for a:link, a:visited, a:active, a:hover.

File: index.html [hunk 4]
```diff
@@ -47,7 +47,7 @@ <h2 class="text-center"><strong>Latest Updates</strong></h2>
     <div class="col-12 col-md-5 col-lg-4 mar-top-20 twitter-roll">
       <a class="twitter-timeline"
         href="https://twitter.com/pysangamam?ref_src=twsrc%5Etfw"
-        data-width="400"
+        data-width="520"
```
Comment (astronomersiva) [inline index.html:50]: I will fix this myself. Let's address only hyperlinks in this PR.

**Review by astronomersiva — CHANGES_REQUESTED** [review]

**Comment by shrayasr** [comment]
Hi @syedjafer have you seen this? Could you please address the comments?

File: css/pysangamam.css (hunk 2 above)
Comment (syedjafer) [reply css/pysangamam.css:21]: In Mobil View , the word Pysangamam is out of screen . So I tried to reduce it .

**Comment by shrayasr** [comment]
@astronomersiva what's the status on this? been pending for a long while now

**Comment by shrayasr** [comment]
Closing this because of inactivity

**Pull request closed without merging**